src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -14,6 +18,39 @@ class Tribe__Tickets__Repositories__Attendee__Commerce extends Tribe__Tickets__A
 	 */
 	protected $key_name = 'tribe-commerce';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.commerce.paypal' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it in a PHP <7.0 compatible way.
+		$attendee_provider = $this->attendee_provider;
+
+		// Add object specific aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
```

Comment:
The same comment about having a getter method for the default args with `wp_parse_args`

Reply:
Going to hold off on changes there per https://github.com/moderntribe/event-tickets/pull/1909#discussion_r544293264